packager/VisibilityChecker.cc

```diff
@@ -192,9 +204,27 @@ class PropagateVisibility final {
                 e.addErrorLine(sym.loc(ctx), "Defined here");
             }
         }
+
+        // If sym is an enum value, it can't be exported directly. Instead, its wrapping enum class must be exported.
+        auto enumClass = packager::getEnumClassForEnumValue(ctx.state, sym);
```

Comment:
You're already in the `packager` namespace so you don't need that prefix if you don't want:

```suggestion
        auto enumClass = getEnumClassForEnumValue(ctx.state, sym);
```